payments_service/auth/tests/test_auth.py

```diff
@@ -0,0 +1,53 @@
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from mock import Mock
+from nose.tools import eq_
+
+from payments_service.base.tests import TestCase, WithDynamicEndpoints
+
+from .. import SessionUserAuthentication
+
+
+class TestSessionUserAuth(TestCase):
+
+    def authenticate(self, **session_kw):
+        request = Mock()
+        request.session = {}
+        request.session.update(session_kw)
+        return SessionUserAuthentication().authenticate(request)
```

Comment:
Session is all local right, so we don't need to mock out the session for network reasons?